### ISSUE TITLE
Incorrect signature/content

### ISSUE THREAD

**Issue by argomark** [grafana/grafana-image-renderer#225]
**What happened**: 

I installed grafana-image-renderer @ 2.1.0 on my Mac with `$ grafana-cli plugins install grafana-image-renderer` and restarted the grafana service.

The plugins page says:
```
The following plugins are disabled and not shown in the list below:
grafana-image-renderer [Modified signature] (Valid signature but content has been modified)
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

Have not attempted a clean grafana install anywhere. Could possibly spin up a VM if it can't be reproduced - seems like something that should always fail??

**Anything else we need to know?**:

I completely removed the plugin and installed it again to ensure there weren't any old files left in the plugin directory. 

I will add the output of `find . -type f -exec md5 '{}' \; > md5sum.txt` for you to compare against what it should be.
[md5sum.txt](https://github.com/grafana/grafana-image-renderer/files/6475422/md5sum.txt)

**Environment**:
- Grafana Image Renderer version: 2.1.0
- Grafana version: 7.5.6 (installed via brew)
- Installed plugin or remote renderer service: installed
- OS Grafana Image Renderer is installed on: MacOS 11.2.3
- User OS & Browser: Chrome 90.0.4430.212 (Official Build) (x86_64)
- Others:

**Comment by argomark** [comment]
Linked to #215?

**Comment by UlrichThiess** [comment]
I solved the problem by deleting the file `MANIFEST.TXT` from the plug-in directory and adding the following entry to the Grafana INI file:

`[plugins]`
`allow_loading_unsigned_plugins = grafana-image-renderer`

**Comment by AgnesToulet** [comment]
Hello! The issue comes from how Grafana is verifiying the plugin signature when the plugin contains symbolic links (as this plugin does). A PR is in progress and it should be fixed in Grafana 8.1. You can follow the PR here: https://github.com/grafana/grafana/pull/34434

**Comment by AgnesToulet** [comment]
Closing this at the PR has been merged and this is not an issue that can be handled by the plugin.
